app/src/main/java/com/github/scribeWizTeam/scribewiz/Fragments/ProfilePageFragment.kt

```diff
@@ -141,31 +151,55 @@ class ProfilePageFragment(contentLayoutId: Int) : Fragment(contentLayoutId) {
             Text(
                 text = "My total recordings : $numRecordings",
                 style = MaterialTheme.typography.h4,
-                fontSize = 22.sp,
+                fontSize = 20.sp,
                 modifier = Modifier.align(Alignment.Start)
             )
             Spacer(Modifier.height(20.dp))
 
```

Comment:
Perhaps you can refactor these different composable elements into their own functions so that it is more modular